ietf/settings.py

```diff
@@ -169,8 +169,8 @@
     STATIC_URL = "/static/"
     STATIC_ROOT = os.path.abspath(BASE_DIR + "/../static/")
 else:
-    STATIC_URL = "https://www.ietf.org/lib/dt/%s/"%__version__
-    STATIC_ROOT = "/a/www/www6s/lib/dt/%s/"%__version__
+    STATIC_URL = "https://static.ietf.org/lib/%s/"%__version__
+    STATIC_ROOT = None
```

Comment:
Check this carefully - I've searched through the uses at django/djano and I _think_ this is legitimate, but another careful look is warranted